example/server.c

```diff
@@ -166,8 +171,13 @@ static int ServerInit(struct Server *s,
     memset(s, 0, sizeof *s);
 
     /* Seed the random generator */
+#if defined(_WIN32)
+    /*mingw doesn't implement timespec_get*/
+    srandom(5);
```

Comment:
something like `srand((unsigned)time(NULL));` ?

Reply:
Agree, missed this thing.